spec/spec_helper_acceptance.rb

```diff
@@ -11,7 +11,7 @@ def pe_install?
 end
 
 run_puppet_install_helper
-install_bolt_on(hosts) unless pe_install?
+install_bolt_on(hosts) unless pe_install? && puppet_version =~ %r{(5\.\d\.\d)}
```

Comment:
This may not be the behaviour you want

This will install bolt if this is a PE install but the puppet version is not 5.x

Reply:
I think the original `unless` is the most valid.  Install bolt if we're not installing PE

Reply:
Thanks Glenn, I've reverted the version check from the bolt install.